src/js/helpers/loginHelper.js

```diff
@@ -32,11 +32,9 @@ export const fetchActiveUser = () => {
 
 	                // check to see if the user is an admin
 	                let isAdmin = false;
-	                res.body.permissions.forEach(function(perm) {
-	                    if (perm == 1) {
-	                        isAdmin = true;
-	                    }
-	                });
+	                if (res.body.permission == 4) {
```

Comment:
Is the API response going to change to a single `permission` value instead of returning an array?